src/middlewared/middlewared/etc_files/local/nut/ups.conf

```diff
@@ -1,8 +1,13 @@
 <%
 	ups_config = middleware.call_sync('ups.config')
-	driver = ups_config['driver'].strip()
+
+	driver = ups_config['driver'].split('$')[0]
+	driver = driver.split('(')[0]  # "blazer_usb (USB ID 0665:5161)"
+	driver = driver.split(' or ')[0]  # "blazer_ser or blazer_usb"
```

Comment:
@themylogin here after split

Reply:
`ups_config['driver']` will never contain string with `\n`, why?

Reply:
`pp(list(map(lambda i:i.split('$')[0].split('(')[0].split(' or ')[0].replace(' ', '\n'), k)))` where k is list of keys from existing ups driver choices. These are one of the few with a space at the end
```
'blazer_usb\n',
 'blazer_usb\n',
 'richcomm_usb\n',
 'blazer_usb\n',
```

Reply:
`pp` is pprint